packages/react/src/components/PageHeader/PageHeader.stories.js

```diff
@@ -106,44 +141,51 @@ const breadcrumbContentActions = (
 );
 
 export const Default = (args) => (
-  <PageHeader.Root>
-    <PageHeader.BreadcrumbBar
-      border={args.border}
-      pageActionsFlush={args.pageActionsFlush}
-      contentActionsFlush={args.contentActionsFlush}
-      renderIcon={args.renderBreadcrumbIcon ? BreadcrumbBeeIcon : null}
-      contentActions={breadcrumbContentActions}
-      pageActions={breadcrumbPageActions}>
-      <Breadcrumb>
-        <BreadcrumbItem href="/#">Breadcrumb 1</BreadcrumbItem>
-        <BreadcrumbItem href="#">Breadcrumb 2</BreadcrumbItem>
-      </Breadcrumb>
-    </PageHeader.BreadcrumbBar>
-    <PageHeader.Content title={args.title}>
-      <PageHeader.ContentText subtitle="Subtitle">
-        Neque massa fames auctor maecenas leo. Mollis vehicula per, est justo.
-        Massa elementum class enim malesuada lacinia hendrerit enim erat
-        pellentesque. Sapien arcu lobortis est erat arcu nibh vehicula congue.
-        Nisi molestie primis lorem nascetur sem metus mattis etiam scelerisque.
-      </PageHeader.ContentText>
-    </PageHeader.Content>
-    <PageHeader.TabBar>
-      <PageHeader.Tabs>
+  <Tabs>
+    <PageHeader.Root>
+      <PageHeader.BreadcrumbBar
+        border={args.border}
+        pageActionsFlush={args.pageActionsFlush}
+        contentActionsFlush={args.contentActionsFlush}
+        renderIcon={args.renderBreadcrumbIcon ? BreadcrumbBeeIcon : null}
+        contentActions={breadcrumbContentActions}
+        pageActions={breadcrumbPageActions}>
+        <Breadcrumb>
+          <BreadcrumbItem href="/#">Breadcrumb 1</BreadcrumbItem>
+          <BreadcrumbItem href="#">Breadcrumb 2</BreadcrumbItem>
+        </Breadcrumb>
+      </PageHeader.BreadcrumbBar>
+      <PageHeader.Content title={args.title}>
+        <PageHeader.ContentText subtitle="Subtitle">
+          Neque massa fames auctor maecenas leo. Mollis vehicula per, est justo.
+          Massa elementum class enim malesuada lacinia hendrerit enim erat
+          pellentesque. Sapien arcu lobortis est erat arcu nibh vehicula congue.
+          Nisi molestie primis lorem nascetur sem metus mattis etiam
+          scelerisque.
+        </PageHeader.ContentText>
+      </PageHeader.Content>
+      <PageHeader.TabBar tags={tags}>
```

Comment:
Maybe for the `Default` story we can keep it without the tags, and display in the `TabBarWithTabsAndTags` story instead?

Reply:
I guess then do we need the `TabBarWithTabsAndTags` story variation then if we're already displaying everything Tabs in the `Default` story?